.gitlab/pipeline/build.yml

```diff
@@ -125,7 +125,7 @@ build-rustdoc:
         find "$path" -name '*.html' | xargs -I {} -P "$(nproc)" bash -c 'process_file "$@"' _ {}
       }
       inject_simple_analytics "./crate-docs"
-    - echo "<meta http-equiv=refresh content=0;url=polkadot_service/index.html>" > ./crate-docs/index.html
+    - echo "<meta http-equiv=refresh content=0;url=developer_hub/index.html>" > ./crate-docs/index.html
```

Comment:
that def needs to be updated, @juangirini worth doing a global search / grep of the workspace for developer_hub before merging this if you haven't already

Reply:
sorry guys, there's been some rebase conflicts that broke the PR after changing its base to master, I'll try to fix this PR, otherwise I'll close it and open a new one